src/marqo/tensor_search/api.py

```diff
@@ -207,6 +208,17 @@ def add_or_replace_documents(
         )
 
 
+@app.post("/indexes/{index_name}/documents/update")
+@throttle(RequestType.UPDATE)
```

Comment:
Partial update might be a better type in case we want to throttle standard update separately later